chapter07/sxg/ex_7_6.pl

```diff
@@ -0,0 +1,27 @@
+#!perl
+use warnings;
+use strict;
+
+my $findW=0;
+my $findF=0;
+my $line;
+
+print "I found:\n";
+while (<>) {
+	$line = $_;
+
+  	if (/wilma/) {
+  		$findW = 1;
+	}
+  	if (/fred/) {
+  		$findF = 1;
+	}
+	if($findF && $findW){
+		print "$line";	
+	}

```

Comment:
Yeah, it would be useful to reset <code>findF</code> and <code>findW</code> at start/end of the loop.